File: wiz-cross-check-java/src/main/java/com/example/wizcrosscheckjava/domain/settlement/PgSettlementHistory.java
package com.example.wizcrosscheckjava.domain.settlement;

import java.time.LocalDate;
import java.time.LocalDateTime;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.Id;
import lombok.AccessLevel;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@Entity
@NoArgsConstructor(access = AccessLevel.PROTECTED)
public class PgSettlementHistory {

	@Id @GeneratedValue
	@Column(name = "pg_settlement_history_id")
	private Long id;

	private String pgName;
	private Long amount;
	private LocalDate settlementDate;
	private final LocalDateTime createdAt = LocalDateTime.now();

	@Builder
	public PgSettlementHistory(Long id, String pgName, Long amount, LocalDate settlementDate) {
		this.id = id;
		this.pgName = pgName;
		this.amount = amount;
		this.settlementDate = settlementDate;
	}
}

File: wiz-cross-check/src/main/kotlin/com/example/wizcrosscheck/domain/settlement/PgSettlementHistory.kt
package com.example.wizcrosscheck.domain.settlement

import jakarta.persistence.*
import java.time.LocalDate
import java.time.LocalDateTime

@Entity
@Table(name = "pg_settlement_history")
class PgSettlementHistory(

    val pgName: String = "테스트용 임의의 PG사",
    val amount: Long,
    val settlementDate: LocalDate,
    val createdAt: LocalDateTime = LocalDateTime.now()
) {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "pg_settlement_history_id")
    var id: Long? = null
}